site/app/models/gradeable/GradedGradeable.php

```diff
@@ -224,6 +230,21 @@ public function getLateDayException($user = null) {
         return $this->late_day_exceptions[$user->getId()] ?? 0;
     }
 
+    /**
+     * Gets the reason of exception for a user
+     * @param User|null $user The user to get exception info for (can be null if not team assignment)
+     * @return string the reason for a user's excused absence extension
+     */
+    public function getReasonForException($user = null) {
```

Comment:
It would be good to change this to a native PHP type declaration.

Reply:
I'm not quite sure what this means could you explain?